server/src/projects/projects.controller.ts

```diff
@@ -58,7 +58,7 @@ export class ProjectsController {
   // Serializes an array of objects into a JSON:API document
   serialize(records, opts?: object): Serializer {
     const ProjectsSerializer = new Serializer('projects', {
-      attributes: ['dcp_projectname', 'dcp_name', 'statecode', 'dcp_visibility', '_dcp_applicant_customer_value', 'dcp_dcp_project_dcp_projectapplicant_Project', 'dcp_dcp_project_dcp_package_project'],
+      attributes: ['dcp_projectname', 'dcp_name', 'statecode', 'statuscode', 'dcp_visibility', '_dcp_applicant_customer_value', 'dcp_dcp_project_dcp_projectapplicant_Project', 'dcp_dcp_project_dcp_package_project'],
```

Comment:
No idea what to do when/if this list gets "too" long (in terms of just visually how it appears in the text editor). I don't think it's too long now (i don't even think our linter complains about that stuff).

Reply:
@allthesignals I had originally autoformatted this with Prettier and it picked this up and made it more readable -- I pulled that back out to make the git diff easier to read in this PR, but take a look at this commit to see how Prettier would make our code more readable, and take away some of these judgement calls about spacing/formatting with some very (imo) sensible default about what makes code readable.
https://github.com/NYCPlanning/labs-applicant-portal/pull/96/commits/16ba20f0e3d4d758c1e2a24f398c1bc19add1dd6